client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1986,6 +1987,13 @@
                         }
 
                         _.extend(item, updates);
+
+                        if (item.profile) {
+                            content.getType(item.profile)
+                            .then(function(type) {
+                                scope.contentType = type;
```

Comment:
so I've commented it on other PR - here you can assign scope.schema and use it in template, this way there can be some default schema in case there is no content type for profile and it would still make sense